src/app-locator.js

```diff
@@ -37,22 +38,25 @@ function getAppId(app) {
 }
 
 function getAppUrl(appId, location = '') {
-  // get the default URL for this appId (if location is local)
-  if ((!location || location === 'local') && DEFAULT_LOCAL_URLS[appId]) {
-    return DEFAULT_LOCAL_URLS[appId]
+  // Probably a valid port, default to localhost.
+  if (/^[0-9]+$/.test(location)) {
+    return `http://localhost:${location}/`
+  }
+
+  // Get the default URL for this appId (when location=local).
+  if (!location || location === 'local') {
```

Comment:
Hmm wondering about this; right now, this would force _all_ apps to use the default local override even if they're not part of the list.

I think we should still be checking that `DEFAULT_LOCAL_URLS[appId]` exists?

Reply:
mmm we don’t have an `appId` at this stage? We are only getting the list of defined app locators. Then in [`appBaseUrl()`](https://github.com/aragon/aragon/blob/71cc90c052c9b735ad719e3c0fe14b0c01843478/src/url-utils.js#L32-L35), we only override if the app ID exists in that object.